crates/relayer/src/chain/cosmos.rs

```diff
@@ -122,9 +122,18 @@ pub mod types;
 pub mod version;
 pub mod wait;
 
-/// fraction of the maximum block size defined in the Tendermint core consensus parameters.
-pub const GENESIS_MAX_BYTES_MAX_FRACTION: f64 = 0.9;
-// https://github.com/cosmos/cosmos-sdk/blob/v0.44.0/types/errors/errors.go#L115-L117
+/// Defines an upper limit on how large any transaction can be.
+/// This upper limit is defined as a fraction relative to the block
+/// maximum bytes. For example if the fraction is `0.9`, then
+/// `max_tx_size` will not be allowed to exceed 0.9 of the
+/// maximum block size of any Cosmos SDK network.
+///
+/// The [default max. block size in Tendermint 0.37 is 21MB](tm-37-max).
+/// With a fraction of `0.9`, then Hermes will never permit the configuration
+/// of `max_tx_size` to exceed ~18.9MB.
+///
+/// [tm-37-max]: https://github.com/tendermint/tendermint/blob/v0.37.0-rc1/types/params.go#L79
+pub const BLOCK_MAX_BYTES_MAX_FRACTION: f64 = 0.9;
```

Comment:
Could we add some rationale for why 0.9 is the default value?

Reply:
d56adcf369d96ce1e4d1e8194d221aceda26f750